<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createPost</class>
 <widget class="QWidget" name="createPost">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="profilePhoto_label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="name_label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="tag_comboBox">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AI</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fun</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Computer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mobile</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Intenet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Engineering</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Travel</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Diplomacy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Technology</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Economics</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Civil</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Architect</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>News</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="postText_textEdit">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>Try to write something...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="media_groupBox">
     <property name="title">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="media_pushButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>media</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="emoji_pushButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>emoji</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="schedule_pushButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>schedule</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="post_pushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>post</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
